designdocs/permissions.md

```diff
@@ -0,0 +1,87 @@
+# proposal for permissions request
+
+
+### Philosophy
+The user-agent will always be closer to the user than a spec. The user agent has greater access to current conditions than we (the spec writers) ever will. As a philosophy we should give the user agent *as much freedom as possible to “do the right thing”* under whatever circumstances are at the time a web application is started.
+
+### scope
+
+Currently we are having difficulties finding a solution that all parties can agree to. This is about the permissions for various properties in WebXR.  This proposal addresses the following concerns
+
+* what permissions are available to be asked for
+* how granular are these permissions
+* are the permissions granted at page load / application start
+* or are the permissions granted at the time of use.
+
+### stating concerns
+
+Concerns about granting at page load time are
```

Comment:
I can change references to 'page load time' to 'request immersive session time'. Would that work for you?

Reply:
Actually, since we'd like it to be possible for a page to request a "local" or "local-floor" reference space for an inline session, this will need to apply more generically.

Reply:
what is the appropriate language here?